src/web/lib/hakuneko/frontend@classic-dark/pages.html

```diff
@@ -514,18 +518,32 @@
                 this.zoom( ( scale < 25 ? 25 : scale ) );
             }
 
+            /**
+             *
+             */
+            defaultZoom( event ) {
+                this.set( 'imageWidth', 75);
+            }
+
+            /**
+             *
+             */
+            fullWidthZoom( event ) {
+                 this.set( 'imageWidth', 100);
+            }
+
             /**
              *
              */
             zoom( scale ) {
-                let previousHeight = this.$.container.scrollHeight;
-                let previousOffset = this.$.container.scrollTop;
-                this.set( 'imageWidth', scale );
-                // embed in timeout function to ensure layout is updated before adjusting offsetY
-                //setTimeout( function() {
-                    // adjust new offsetY depending on height changed ratio of container after scaling images
-                    this.$.container.scrollTop = previousOffset * this.$.container.scrollHeight / previousHeight;
-                //}.bind( this ), 0 );
+                    let previousHeight = this.$.container.scrollHeight;
```

Comment:
what with this over spacing (2 tabs instead of 1 ?) Remove them ;)

Reply:
The extra tab has now been removed